openwisp_controller/geo/base/models.py

```diff
@@ -35,6 +38,20 @@ def clean(self):
             )
         return super().clean()
 
+    def save(self, *args, _set_estimated=False, **kwargs):
+        # estimate locations are created only via `manage_estimated_locations` task
+        # so we set `is_estimated` to False from all other sources as they imply
+        # manual refinement
+        if not _set_estimated:
```

Comment:
what if `self.is_estimated` is already `False`? Can this happen?

Reply:
The default value of `is_estimated` is `False`. 
I think we might have to set all is_estimated to Null when the feature is disabled for the organization. This is because if there are estimated locations for an org and the feature is disabled then whenever the location is saved from admin then the following validation will give error:
https://github.com/openwisp/openwisp-controller/blob/889dde357e8877ddf9b03e9f54f0512f0422397f/openwisp_controller/geo/base/models.py#L26

Reply:
Have done this in https://github.com/openwisp/openwisp-controller/pull/1088/commits/77fc764c187282a49c8277ad9c284a5407901aa4